lms/djangoapps/email_marketing/signals.py

```diff
@@ -54,18 +62,15 @@ def add_email_marketing_cookies(sender, response=None, user=None,
         if sailthru_content:
             post_parms['cookies'] = {'anonymous_interest': sailthru_content}
 
-    try:
-        sailthru_client = SailthruClient(email_config.sailthru_key, email_config.sailthru_secret)
-        log.info(
-            'Sending to Sailthru the user interest cookie [%s] for user [%s]',
-            post_parms.get('cookies', ''),
-            user.email
-        )
-        time_before_call = datetime.datetime.now()
+    time_before_call = datetime.datetime.now()
+    sailthru_response = get_email_cookies_via_sailthru.delay(post_parms)
 
-        sailthru_response = \
-            sailthru_client.api_post("user", post_parms)
+    try:
+        sailthru_response.get(timeout=3, propagate=True)
```

Comment:
Is this 3 seconds? Or 3 milliseconds?
Can you use a constant here instead of just the number to make this more maintainable?